### PR TITLE
TT-1689 move tests in tests folder

### PR DESCRIPTION
**Pull request by sdonk**
To do (delete all that do not apply):

 - [x] Change has a jira ticket that has the correct status.
 - [x] Changelog entry added.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -19,9 +19,9 @@
 from core.models import RoutingSettings
 from groups.models import GroupInfo
 from users.models import UserProfile
-from great_international.tests.factories import InternationalHomePageFactory
-from find_a_supplier.tests.factories import IndustryPageFactory
-from users.tests.factories import UserFactory
+from .great_international.factories import InternationalHomePageFactory
+from .find_a_supplier.factories import IndustryPageFactory
+from .users.factories import UserFactory
```
Comment (MichaelWalker) [inline tests/conftest.py]: I guess these must work otherwise the tests would fail...
Is the `.` doing something special or it the same as without it?

Comment (sdonk) [reply tests/conftest.py]: It's looking into the same folder (`tests`). It's a shortcut for `from tests.great_international.factories import InternationalHomePageFactory`

Comment (eadpearce) [reply tests/conftest.py]: it's just a relative import as opposed to an absolute import

Comment (MichaelWalker) [reply tests/conftest.py]: Cool - I actually didn't know python had an explicit syntax for those! Nice!

(I thought it just scanned through its list of directories, and the current was always high enough on the list that everything works fine without the dot! Much nicer to be explicit about it though!)

**Pull request merged**